x-pack/packages/kbn-cloud-security-posture/storybook/config/kibana.jsonc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Does this need to be a package for storybook to work? Or will this be imported somewhere else?

Reply:
Just for storybook to work